### ISSUE TITLE
SLIMIT returns results from several series

### ISSUE THREAD

**Issue by brandonbasker** [influxdata/influxdb#7764]
### Bug report

__System info:__ influxdb 1.1.1 running on a debian-jessie container

When i run the following query this happens:

```
> select * from traffic group by * SLIMIT 1
name: traffic
tags: dst=10.10.10.202, id=099158c2-de34-4289-88f4-9133b1c2d434, port=34516, protocol=UDP, src=10.10.10.82
time                    b_a_len b_p_count     max                     min                     a_b_len a_p_count
----                    ------- -----------     ---                     ---                     ------- -----------
1482613500825808942     0       0               1482613500825808942     1482613500825808942     125     1


name: traffic
tags: dst=10.10.10.202, id=19f860e5-306d-490d-a995-a7c4ad58722d, port=41339, protocol=UDP, src=10.10.10.82
time                    b_a_len b_p_count     max                     min                     a_b_len a_p_count
----                    ------- -----------     ---                     ---                     ------- -----------
1482656699672247437     0       0               1482656699672247437     1482656699672247437     125     1


name: traffic
tags: dst=10.10.10.202, id=26420b8f-75ee-4d1d-9ab7-dc4a21028d48, port=44666, protocol=UDP, src=10.10.11.33
time                    b_a_len b_p_count     max                     min                     a_b_len a_p_count
----                    ------- -----------     ---                     ---                     ------- -----------
1482444523876318069     0       0               1482444523876318069     1482444523876245008     232     2


name: traffic
tags: dst=10.10.10.202, id=3e4f9d8b-2a20-4622-9fb9-e64c19298661, port=35823, protocol=UDP, src=10.10.10.81
time                    b_a_len b_p_count     max                     min                     a_b_len a_p_count
----                    ------- -----------     ---                     ---                     ------- -----------
1482470314879782279     0       0               1482470314879782279     1482470314879782279     135     1

> show series where id='099158c2-de34-4289-88f4-9133b1c2d434'
key
traffic,dst=10.10.10.202,id=099158c2-de34-4289-88f4-9133b1c2d434,port=34516,protocol=UDP,src=10.10.10.82

> show series where id='19f860e5-306d-490d-a995-a7c4ad58722d'
key
traffic,dst=10.10.10.202,id=19f860e5-306d-490d-a995-a7c4ad58722d,port=41339,protocol=UDP,src=10.10.10.82
```
Although each point is part of a different series they are somehow returned together. All points are using a single default RP.
I don't really have a good guide as to how to reproduce this, and when manually entering the points the query is correct.

**Comment by desa** [comment]
@brandonbasker Would be great if we could get repro steps. How are you writing the data?

**Comment by brandonbasker** [comment]
@desa I write using the the golang client library. I create a batch point and write about ~100 points per minute into the traffic measurement. After about a day of running i had 5000 series. I think the problem might be related to the number of series growing relatively fast. I completely changed the way i save data so it seems i found away around this, but it still seems like a bug.

**Comment by desa** [comment]
@brandonbasker any chance you could share some of the code so we can try to reproduce it?

**Comment by Nicholi** [comment]
I think I've just noticed a similar issue on my end. Querying with `SLIMIT 1` and I am getting 2 series in result (`SLIMIT 2` = 3 series, `SLIMIT 3` = 3 series, and then its normal after that). My query is somewhat more complex than @brandonbasker (but my data is not rapidly changing). I am using some aggregates along with a WHERE clause and fill(0). I noticed if I remove my WHERE clause SLIMIT seems to be obeyed again.

I'll see about sanitizing my data and making a small dump similar to the NOAA water db.

influxdb v1.2.4
